changelogs/unreleased/gh-8817-bug-fix-set-language-shared-between-conn-clients.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/box
+
+* Fixed a bug in the box console implementation because of
+  which the `language` parameter was shared between connected
```

Comment:
Nit: current length of lines in commit message is 55, max allowed length is 72 symbols [^1]. I propose to reformat it.

[^1]: https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message

Reply:
Done